### PR TITLE
Bugfix: Some objects have invalid scales when replay rendering

### PR DESCRIPTION
**Pull request by 0mdc**
## Motivation and Context

When replay rendering (from the classic replay renderer, batch renderer, Unity or Blender), some objects have incorrect scaling.

In Habitat, the scale of nodes comes from two sources:
* The `CreationInfo scale`, which is typically user-defined.
* The `asset scale`. GLTF files (and other complex formats) are composed of nodes that can be scaled. Those are artist-defined.
All importers will automatically apply the `asset scale` when loading a model.

To obtain the desired scale, we must multiply the `CreationInfo scale` with `asset scale`.

Because the recorder is writing the final scale (`CreationInfo scale` * `asset scale`) into keyframes, the renderers apply the `asset scale` twice when loading assets.

This changeset fixes replay renderer scaling by ensuring that only the `CreationInfo scale` is recorded, as the `asset scale` is already defined in the model.

**Note:** Upon importing URDF files, the graphics nodes were scaled directly rather than passing the information through `CreationInfo`. This would "hide" URDF scaling from gfx-replay.

**The figure below shows results in Unity.** Note that the behavior is the same in Blender, classic and batch replay renderers.
| Before  | After |
| ------------- | ------------- |
|scale_before|scale_after|

## How Has This Been Tested

Tested locally on classic and Unity replay renderers.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by aclegg3** [comment]
Marge `main` to pass CI :slightly_smiling_face:

File: src/esp/physics/bullet/BulletPhysicsManager.cpp [hunk 1]
```diff
@@ -296,14 +296,14 @@ bool BulletPhysicsManager::attachLinkGeometry(
           Mn::Color4(material->m_matColor.m_specularColor);
     }
 
+    auto scale = Mn::Vector3{1.0f, 1.0f, 1.0f};
```
[the remark this answers is not in the document]
Comment (0mdc) [reply src/esp/physics/bullet/BulletPhysicsManager.cpp:299]: The global scale is already applied.

`Model::setGlobalScaling` in `URDFParser.cpp` sets `m_meshScale`, `m_boxSize`, `m_sphereRadius`, etc., which are applied below.

It's a bit confusing though, as `GlobalScale/ing` and `UniformScale/ing` names are used interchangeably but aren't directly applied.

**Comment by 0mdc** [comment]
> Is there any testing for this? I know there are replay tests, so maybe something that instances a scene and saves a replay, gets an image, loads the replay and compares a new images to the saved image? Could catch many bugs with replay down the line.

I added a visual test for some edge cases.

**Review by aclegg3 — APPROVED** [review]
LGTM once tests pass :)

**Review by jturner65 — APPROVED** [review]
